### PR TITLE
Feat/chal 21/create music module

### PR DESCRIPTION
**Pull request by tojvex**

### CODE REVIEW

File: src/music/music.repository.ts [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+@Injectable()
+export class MusicRepository {
+    constructor(@InjectRepository (Music)
+private MusicRepository: Repository <Music>) {}
+  async create(createMusicDto: CreateMusicDto) {
+    
+    const music =  await this.MusicRepository
+    .createQueryBuilder()
+    .insert()
+    .into(Music)
```
Comment (1Lazare) [inline src/music/music.repository.ts]: When you describe the repository in the controller it is no longer necessary to use into

File: src/music/music.repository.ts [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+    .values(createMusicDto)
+    .execute()
+
+    return music.generatedMaps[0]
+    
+  }
+
+  findAll() {
```
Comment (1Lazare) [inline src/music/music.repository.ts]: change functions to async this will return promise

File: src/music/music.repository.ts [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+    return this.MusicRepository
+    .createQueryBuilder()
+    .getMany()
+  }
+
+  findOne(id: number) {
```
Comment (1Lazare) [inline src/music/music.repository.ts]: same change everywhere to async functions

File: src/music/music.service.ts [hunk 4]
```diff
@@ -0,0 +1,28 @@
+import { Injectable } from '@nestjs/common';
+import { CreateMusicDto } from './dto/create-music.dto';
+import { UpdateMusicDto } from './dto/update-music.dto';
+import { MusicRepository } from './music.repository';
+
+@Injectable()
+export class MusicService {
+  constructor(private readonly MusicRepository:MusicRepository) {}
+  create(createMusicDto: CreateMusicDto) {
```
Comment (1Lazare) [inline src/music/music.service.ts]: same change to async functions

File: src/music/entities/music.entity.ts [hunk 5]
```diff
@@ -0,0 +1,30 @@
+import { Column, CreateDateColumn, DeleteDateColumn, Entity, PrimaryGeneratedColumn, UpdateDateColumn } from "typeorm";
+
+@Entity()
+export class Music {
```
Comment (1Lazare) [inline src/music/entities/music.entity.ts]: change name to MusicEntity

File: src/music/music.repository.ts (hunk 1 above)
Comment (tojvex) [reply src/music/music.repository.ts]: Done

File: src/music/music.repository.ts (hunk 2 above)
Comment (tojvex) [reply src/music/music.repository.ts]: Done

File: src/music/music.repository.ts (hunk 3 above)
Comment (tojvex) [reply src/music/music.repository.ts]: Done

File: src/music/music.service.ts (hunk 4 above)
Comment (tojvex) [reply src/music/music.service.ts]: Done

File: src/music/entities/music.entity.ts (hunk 5 above)
Comment (tojvex) [reply src/music/entities/music.entity.ts]: Done

**Pull request merged**